debian/changelog

```diff
@@ -1,4 +1,6 @@
   [Michele Simionato]
+  * Now the URL /v1/calc/list returns a list of dictionaries with a boolean
+    key is_running
```

Comment:
Maybe is better something like: a new key ```is_running``` is added to the list of dictionaries returned by ...
to emphasize the new part in a more clear way.